about.html

```diff
@@ -0,0 +1,115 @@
+                </li>
+            </ul>
+        </div>
+    </nav>
+ 
+    <main class="container my-5">
+      
+        <section class="row">
+            <div class="col-md-4">
+                <img src="images/photoprofil3.jpg" alt="Photo de Gabriel Gagné" class="img-fluid rounded">
+            </div>
+            <div class="col-md-8">
+                <h2>Qui suis-je ?</h2>
+                <p>Je suis un jeune passionné de programmation, de musique, de sport et de gaming. J'aime faire de
+                    nouvelles rencontres et je suis très sociable. Je suis actuellement en train d'étudier la
+                    programmation et je suis à la recherche d'un emploi dans une entreprise où je pourrais mettre à
+                    profit mes compétences et participer activement à son succès.</p>
+            </div>
+        </section>
+      
+        <section class="row mt-5">
+            <div class="col-md-12">
+                <h2>CV</h2>
+                <p><strong>Objectif de travail :</strong></p>
+                <address>
+                    144 rang des Moulanges<br>
+                    St-Apollinaire (Qc)<br>
+                    G0S 2E0<br>
+                    Tél. : 418-665-1307<br>
+                    Courriel : gabrielgagne17@hotmail.com
+                </address>
+                <p>Travailler dans une entreprise où mes aptitudes seront mises à profit et où je pourrai tirer
+                    avantages de mes connaissances, tout en participant de façon active, responsable et ponctuel au
+                    succès de l’entreprise.</p>
+                <p><strong>Études :</strong></p>
+                <ul>
+                    <li>École Secondaire Du Plateau 2010 à 2015</li>
+                    <li>Diplôme d’études secondaire</li>
+                    <li>Sports-Études Hockey</li>
+                </ul>
+                <p><strong>Formation :</strong></p>
+                <ul>
+                    <li>Chariot élévateur (cariste)</li>
+                </ul>
+                <p><strong>Expériences de travail :</strong></p>
+                <ul>
+                    <li>Ouvrier d’usine pour la compagnie Valconcept inc. 2015 à auj. (Temps plein)</li>
+                    <li>(entreprise manufacturière de portes et fenêtres) 2011 à 2015 (Temps partiel)</li>
+                </ul>
+            </div>
+            <div class="col-lg-6">
+                <h2>Intérêts et loisirs</h2>
+                <ul>
+                    <li>Sports (en général)</li>
+                    <li>Musique / Lecture</li>
+                    <li>Cinématographie</li>
+                </ul>
+            </div>
+            </div>
+            <div class="row">
+                <div class="col-lg-12">
+                    <h2>Références</h2>
+                    <p>Peuvent être fournies sur demande.</p>
+                </div>
+            </div>
+            </div>
+</main>
+
```

Comment:
Please, use Visual Studio Code to format your page. This page need a better indentation.